apps/typeform/lambda/app.js

```diff
@@ -15,6 +15,18 @@ const app = express();
 
 const FRONTEND = path.dirname(require.resolve('@contentful/typeform-frontend'));
 
+const computeLastModifiedTime = () => {
+  const deployTime = process.env.DEPLOY_TIME_UNIX;
+  if (typeof deployTime === 'undefined') {
+    throw new Error('Missing DEPLOY_TIME_UNIX env var');
```

Comment:
you want to break things or just do "10 minutes ago" as a fallback?

Reply:
this will fail hard and loud when the server starts, which is always my personal prefernce for env vars 🤷